load.Rmd

```diff
@@ -37,7 +37,7 @@ if(!exists("region")) region <- "liverpool-city-region"
 pct_data <- file.path("..", "pct-data")
 pct_bigdata <- file.path("..", "pct-bigdata")
 pct_privatedata <- file.path("..", "pct-privatedata")
-pct_shiny <- file.path("..", "pct-shiny")
+pct_shiny <- file.path("..", "pct-shiny", "regions_www")
```

Comment:
Yes I noticed that - fixed in latest commit which has been tested for both buildmod.R and buildmaster.R